<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSongDialog</class>
 <widget class="QWidget" name="EditSongDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Song Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/openlp-2.qrc">
    <normaloff>:/icon/openlp.org-icon-32.bmp</normaloff>:/icon/openlp.org-icon-32.bmp</iconset>
  </property>
  <widget class="QWidget" name="TopWidget" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QWidget" name="TextWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="TitleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="TitleEditItem">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="AlternativeTitleLabel">
       <property name="text">
        <string>Alternative Title:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="AlternativeEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LyricsLabel">
       <property name="text">
        <string>Lyrics:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QTextEdit" name="LyricsTextEdit">
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="AuthorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>0</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QLabel" name="AuthorsLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>45</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Authors:</string>
     </property>
    </widget>
    <widget class="QWidget" name="AuthorSelectWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>25</y>
       <width>361</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="AuthorSelectWidgetLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="AuthorsSelectionComboItem">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AuthorAddtoSongItem">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add to Song</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddAuthorsButton">
        <property name="text">
         <string>Add Authors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="SongBookGroup" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>85</y>
       <width>351</width>
       <height>27</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="SongBookLayout">
       <item>
        <widget class="QComboBox" name="SongbookCombo"/>
       </item>
       <item>
        <widget class="QPushButton" name="AddSongBookButton">
         <property name="text">
          <string>Add Song Book</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="SelectedAuthorsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>96</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Selected Authors:</string>
     </property>
    </widget>
    <widget class="QWidget" name="CopyrightWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>368</y>
       <width>147</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="CopyrightWidgetLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QLabel" name="SongBookLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>57</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>SongBook</string>
     </property>
    </widget>
    <widget class="QLabel" name="Topiclabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>242</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>SongTopic</string>
     </property>
    </widget>
    <widget class="QLabel" name="SelectedTopicsLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>96</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Selected Topics:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>341</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="SelectedAuthorsGroup">
      <item>
       <widget class="QTableWidget" name="AuthorsListView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="ToticsListView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>130</y>
       <width>341</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="SongTopicLayout">
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="AddTopicsToSongButton">
        <property name="text">
         <string>Add to Song</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddTopicButton">
        <property name="text">
         <string>Add Topic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>341</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="RemoveAuthorsLayout">
      <item>
       <widget class="QPushButton" name="AuthorRemoveItem">
        <property name="text">
         <string>Remove Selected Author(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Remove Selected Topics</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="ThemeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>641</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="ThemeLabel">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ThemeSelectionComboItem"/>
    </item>
    <item>
     <widget class="QPushButton" name="ThemeAddItem">
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Add a Theme</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="ButtonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>570</y>
     <width>156</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="VerseOrderLayout">
    <item>
     <widget class="QLabel" name="VerseOrderLabel">
      <property name="text">
       <string>Verse Order:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="VerseOrderEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>461</y>
     <width>371</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="CopyrightLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="CopyrightInfoLabel">
      <property name="text">
       <string>Copyright Info:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="CopyrightEditItem"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="CopyrightInsertItem">
      <property name="maximumSize">
       <size>
        <width>29</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Â©</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>TopWidget</zorder>
  <zorder>ThemeWidget</zorder>
  <zorder>ButtonBox</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources>
  <include location="../images/openlp-2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSongDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSongDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>-50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
